### ISSUE TITLE
[Bug]: Symlinked Extensions folder breaks the extension

### ISSUE THREAD

**Issue by Lukium** [zero01101/openOutpaint-webUI-extension#41]
### Is ths issue about the extension?

- [X] Yup, this is for sure about the extension

### What happened?

I run a community server with multiple instances of WebUI each on its own dedicated RTX 3090. As a result, to avoid wasting a ton of space duplicating things that are used by every instance, most of the folders are symlinked and shared, including extensions, models, checkpoints, repositories, embeddings, etc. (anything that can be shared between all instances.)

This is the first instance that I have come across that seems to not be able to be symlinked.

### Steps to reproduce the problem

1. Move extensions folder outside of the webui folder
2. Symlink it back into the folder
3. Run webui and go to Outpaint Tab

You get several instances of:
```
All files must contained within the Gradio python app working directory, or be a temp file created by the Gradio python app.
```

### What should have happened?

It should've just worked

### Commit where the problem happens

8491f85

### What platforms do you use to access openOutpaint?

Windows

### What browsers do you use to access the UI ?

Google Chrome, Microsoft Edge

### Browser Extensions/Addons

Only this one, as I'm putting together a new update for the main servers and testing one extension at a time.

### AUTOMATIC1111 webUI Commandline Arguments

set COMMANDLINE_ARGS=--api --ckpt-dir ".\checkpoints" --opt-split-attention --opt-channelslast --xformers --force-enable-xformers --deepdanbooru --gradio-img2img-tool color-sketch --device-id 0

**Comment by Lukium** [comment]
Hmmm... this actually seems to be a new issue with Automatic1111... I'm not experiencing it with a couple of other extensions... highly annoying...

**Comment by seijihariki** [comment]
Yeah, we actually took steps to allow it to run from symlinked extensions due to docker user complaints.
